examples/arkane/species/C2H4/ethene.log

```diff
@@ -1826,4 +1826,4 @@
  00000900,0.00001502,0.,0.00005012\\\@
  Job cpu time:  0 days  0 hours  0 minutes 39.5 seconds.
  File lengths (MBytes):  RWF=    329 Int=      0 D2E=      0 Chk=     11 Scr=      1
- Normal termination of Gaussian 03 at Fri Feb  9 00:59:20 2007.
+Normal termination of Gaussian 03 at Fri Feb 9 00:59:20 2007.
```

Comment:
Were the spacing changes in this file intentional?

Reply:
No. I might have copy-pasted this file, so perhaps the text editor added them. I'll revert it